src/library/Uno.Cupertino/Styles/Controls/Button.xaml

```diff
@@ -0,0 +1,405 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:toolkit="using:Uno.UI.Toolkit"
+					xmlns:ios="http://uno.ui/ios"
+					xmlns:android="http://uno.ui/android"
+					xmlns:not_android="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:wasm="http://uno.ui/wasm"
+					xmlns:not_win="http://uno.ui/not_win"
+					mc:Ignorable="d ios android wasm not_win">
+
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="../Application/Colors.xaml" />
+		<ResourceDictionary Source="../Application/AnimationConstants.xaml" />
+	</ResourceDictionary.MergedDictionaries>
+
+	<!-- Documented variables -->
+	<Thickness x:Key="CupertinoButtonPadding">20</Thickness>
```

Comment:
Looking at flutter it seems like the Button Padding is 16

https://github.com/flutter/flutter/blob/9ce3f76c5353fa2caa6350adb1a2cc6f65ba92f2/packages/flutter/lib/src/cupertino/button.dart#L13

Reply:
Indeed, but in the Figma it is 20. Let me know which one is better!
https://www.figma.com/file/rm3i2SyeCE1sSZWPPwyDiM/UNO---Cupertino-UI-Kit?node-id=127%3A37